lib/lbt/analyzer/XMLTemplateAnalyzer.js

```diff
@@ -1,19 +1,9 @@
-"use strict";
 
-/*
- * TODOS
- * - find better way to distinguish between aggregation tags and control tags
- *   (currently, existence in pool is used to recognize controls)
- * - support alternative namespace URLs for libraries (as used by XSD files)
```

Comment:
@RandomByte should we keep this todos?

Reply:
Hm, I don't know this module well enough to tell whether the comment is still required.

It got removed with https://github.com/SAP/ui5-builder/commit/587b4c8e2537c2b6d81afc980f8ac1dca0153ca3#diff-7ed80359732764a080aa58a9fd7b3ce1989332b39988be9eb8bfd1835bed0a3b, so maybe @d3xter666 or @matz3 can tell?

Otherwise I would opt to re-add it for the time being, since it might have been removed by mistake.

Reply:
Done